apps/vmq_server/src/vmq_mqtt5_fsm.erl

```diff
@@ -855,7 +855,9 @@ connected(close_timeout, State) ->
     %% As we're in the connected state, it's ok to ignore this timeout
     {State, []};
 connected(Unexpected, State) ->
```

Comment:
Totally a detail: Alternative is to match in function args to get to state variables `connected(Unexpected, #state{subscriber_id = Subscriber} = State) ->`
Both ways fine for me.